mod_cluster/undertow/src/test/java/org/wildfly/mod_cluster/undertow/TestServer.java

```diff
@@ -9,14 +9,23 @@
 import org.wildfly.extension.undertow.Server;
 import org.wildfly.extension.undertow.UndertowService;
 
+import java.util.function.Consumer;
```

Comment:
The import order is wrong, the order we use is static, java, the rest. Please fix that.